src/components/charts/bar-chart/bar-chart.js

```diff
@@ -155,6 +155,13 @@ SimpleBarChart.propTypes = {
   onMouseMove: PropTypes.func,
   forceFixedFormatDecimals: PropTypes.number,
   margin: PropTypes.object,
+  /** Margin of the chart */
+  chartMargin: PropTypes.shape({
```

Comment:
Having two margins (`margin`, `chartMargin`) will be confusing. Funny fact is the I cannot see that `ResponsiveContainer` has `margin` property :flushed: (https://github.com/recharts/recharts/blob/master/src/component/ResponsiveContainer.js). So I would rename `chartMargin` to `margin` and remove the old `margin` property.

Reply:
That could probably "fix" all charts where the margin is not working but it should.